sdk/storage/Azure.Storage.Blobs/src/AppendBlobClient.cs

```diff
@@ -28,10 +28,10 @@ namespace Azure.Storage.Blobs.Specialized
     /// operation.  Updating or deleting of existing blocks is not supported.
     /// Unlike a block blob, an append blob does not expose its block IDs.
     ///
-    /// Each block in an append blob can be a different size, up to a maximum
-    /// of 4 MB, and an append blob can include up to 50,000 blocks.  The
-    /// maximum size of an append blob is therefore slightly more than 195 GB
-    /// (4 MB X 50,000 blocks).
+    /// Each block in an append blob can be a different size.
+    /// Beginning with x-ms-version 2022-11-02, the maximum append size is 100 MB.
+    /// For previous version, the maximum append size is 4 MB.
+    /// Append blobs can include up to 50,000 blocks.
```

Comment:
Can we include a link to the rest docs here since those will always be up to date with the latest limitations